packages/core/src/extensions/Blocks/nodes/BlockContent/ImageBlockContent/uploadToTmpFilesOrg_DEV_ONLY.ts

```diff
@@ -0,0 +1,13 @@
+export const uploadToTmpFilesOrg = async (file: File) => {
```

Comment:
let's rename the function as well? `uploadToTmpFilesDotOrg_DEV_ONLY`?

(added `Dot` for clarity and `DEV_ONLY`)